rider/src/main/java/org/sonarlint/intellij/rider/RiderAnalysisConfigurator.kt

```diff
@@ -37,11 +36,6 @@ class RiderAnalysisConfigurator : AnalysisConfigurator {
         if (monoRuntime != null) {
             result.extraProperties["sonar.cs.internal.monoExeLocation"] = monoRuntime.getMonoExe().absolutePath
         }
-        val solutionFile = module.project.solutionFile
```

Comment:
This removes what was introduced for [SLI-628](https://jira.sonarsource.com/browse/SLI-628), should we remove the ticket from the version ?

Reply:
Oups... not expected